app/presenters/comment_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class CommentPresenter < ApplicationPresenter
+  presents :comment
+  delegate :text, to: :comment
+
+  def submitter_name
+    comment.submitter.name
+  end
+
+  def attribution_link
+    string, link = if h.user_signed_in?
+      ["Reply to #{submitter_name}", h.comment_reply_comment_path(comment)]
+    else
+      [" - contributed by #{submitter_name}", h.user_path(submitter_name)]
+    end
+
+    h.link_to string, link
+  end
+
+  def sum
+    if comment.votes.exists?
+      comment.votes.sum(:value)
+    else
+      0
+    end
```

Comment:
I'm curious if `comment.votes.sum(:value)` will return `0` if no votes exist on the comment, thus eliminating the need for the existence check.

Reply:
indeed - the following is sufficient: `comment.votes.sum(:value)`